docs/components/cursor.md

```diff
@@ -80,6 +80,7 @@ AFRAME.registerComponent('cursor-listener', {
 | downEvents  | Array of additional events on the entity to *listen* to for triggering `mousedown` (e.g., `triggerdown` for vive-controls).  | []                               |
 | fuse        | Whether cursor is fuse-based.                                                                                              | false on desktop, true on mobile |
 | fuseTimeout | How long to wait (in milliseconds) before triggering a fuse-based click event.                                             | 1500                             |
+| mode     | Camera (gaze) or mouse controlled. | gaze
```

Comment:
I think the `mode` sort of makes sense, but I don't really like having to provide strings, that `gaze` isn't the only other way to use the cursor, and that `gaze` is not represented in the code as doing anything.